src/uk/gov/hmcts/cmc/integrationtests/IntegrationTests.groovy

```diff
@@ -83,41 +83,44 @@ class IntegrationTests implements Serializable {
     steps.sh "${dockerComposeCommand()} pull"
   }
 
-  private void startTestEnvironment(String application) {
+  private void startTestEnvironment() {
```

Comment:
It would be great to no start legal-frontend when citizen-frontend triggers IT. The same way it would be great not to start citizen-frontend when legal-frontend triggered IT. Both should be started only when claim-store triggered quick version to the IT.

Could be next step / separate PR as frontend startup time is relatively short.

Reply:
👍